MasterWAMPServer.spec.js

```diff
@@ -1,46 +1,47 @@
-'use strict';
+/* eslint no-unused-expressions: 0 */
+// https://github.com/jonathanglasmeyer/graphql-sequelize/commit/00814cac3aa9fa6d20aed38df838dcbc4b4ab9b4
 
 const chai = require('chai');
 const sinon = require('sinon');
 const MasterWAMPServer = require('./MasterWAMPServer');
 const MasterWAMPRequestSchema = require('./schemas').MasterWAMPRequestSchema;
-const MasterWAMPResponseSchema = require('./schemas').MasterWAMPResponseSchema;
-const expect = chai.expect;
 
-describe('MasterWAMPServer', function () {
+const expect = chai.expect;
 
-	describe('constructor', function () {
+describe('MasterWAMPServer', () => {
+	describe('constructor', () => {
 		let fakeSCServer;
 
-		beforeEach(function () {
+		beforeEach(() => {
 			fakeSCServer = {
 				on: sinon.spy(),
-				sendToWorker: sinon.spy()
+				sendToWorker: sinon.spy(),
 			};
 		});
 
-		it('create SlaveWAMPServer with socketCluster field', function () {
+		it('create SlaveWAMPServer with socketCluster field', () => {
 			const masterWAMPServer = new MasterWAMPServer(fakeSCServer);
 			expect(masterWAMPServer).to.have.property('socketCluster').to.be.a('object');
 		});
 
 
-		it('should start listening on "workerStart"', function () {
+		it('should start listening on "workerStart"', () => {
+			/* eslint-disable no-new */
```

Comment:
skipping the rule should be applied just for tests only. I will add the eslint variable setting at the beginning of tests files then